config/app.php

```diff
@@ -106,7 +106,7 @@
     |
     */
 
-    'faker_locale' => 'en_US',
+    'faker_locale' => 'zh_cn',
```

Comment:
why? Waarom de moeite doen om dit te veranderen en waarom eigenlijk?

Reply:
Extra punten voor creativiteit :'). Maar het was een erg kleine moeite, en de chineze namen maken het in mijn optiek net wat leuker. Overbodig, sure.